src/app/App.js

```diff
@@ -0,0 +1,26 @@
+import './App.css';
+import { Counter } from "../shared/ui/Counter/Counter";
+import { GenreSelect } from "../shared/ui/GenreSelect/GenreSelect";
+import { SearchForm } from "../shared/ui/SearchForm/SearchForm";
+import { generList } from "../shared/ui/GenreSelect/GenerMockData";
+
+
+export const App = () => {
+  const handleSearch = (val) => { };
+
+  const handleChangeSelectedGenre = (val) => { }
```

Comment:
it's better to clarify 
```suggestion
  const handleChangeSelectedGenre = (searchValue) => { }
```

Reply:
fixed